Library/Formula/duck.rb

```diff
@@ -0,0 +1,25 @@
+require "formula"
+
+class Duck < Formula
+  homepage "https://duck.sh/"
+  # Reference to the CLI version of Cyberduck
+  url "https://update.cyberduck.io/cli//duck-4.6.2.16269.tar.gz"
+  version "4.6.2.16269"
+  sha1 "c417af3c8069ac5a3dc58329eaf2323234925cb5"
+
+  def install
+    # Because compiling would need a JDK and xcodebuild we just use the pre-compiled binary.
+    libexec.install Dir['*']
+    bin.install_symlink "#{libexec}/Contents/MacOS/duck" => "duck"
+  end
+
+  test do
+    system "#{bin}/duck", "-version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!